tests/unit/test_api_security.py

```
@@ -76,107 +108,129 @@ def test_rate_limit_exceeded(self):
         allowed, reason, meta = self.rate_limiter.allow_request(client_ip, user_agent)
         self.assertFalse(allowed)
         self.assertEqual(reason, "Abuse detected")
-    
+
+    def test_abuse_detection_reset_after_cooldown(self):
+        """Test that abuse detection resets after cooldown period."""
+        client_ip = "192.168.1.6"
+        user_agent = "test-agent"
+
+        # Simulate rapid-fire requests to trigger abuse detection
+        for i in range(11):  # More than 10 requests in 1 second
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)